Taskfile.yml

```diff
@@ -270,6 +273,7 @@ tasks:
         -hana-url=$FERRETDB_HANA_URL
         -compat-url='mongodb://username:password@127.0.0.1:47018/?tls=true&tlsCertificateKeyFile=../build/certs/client.pem&tlsCaFile=../build/certs/rootCA-cert.pem'
         -disable-filter-pushdown={{.DISABLE_FILTER_PUSHDOWN}}
+        -disable-sort-pushdown={{.DISABLE_SORT_PUSHDOWN}}
```

Comment:
If `-disable-sort-pushdown` is set as false and `-enable-unsafe-sort-pushdown` is set as true, is pushdown enabled?